appserver/monitoring-console/webapp/src/main/webapp/js/monitoring-console.js

```diff
@@ -1322,6 +1349,17 @@ MonitoringConsole.View.Units = (function() {
       tb: 1024 * 1024 * 1024 * 1024,
    };
 
+   /**
+    * Factors by unit
+    */
+   const FACTORS = {
```

Comment:
Duplicit constant definition.  Same is in another file of the same diff (mc-view-units.js). Could they be shared?